.github/workflows/operate-ci-build-reusable.yml

```diff
@@ -146,3 +147,12 @@ jobs:
         if: ${{ env.IS_DEFAULT_BRANCH == 'true' }}
         run: |
           mvn -f operate org.sonatype.plugins:nexus-staging-maven-plugin:deploy-staged -DskipTests=true -P -docker,skipFrontendBuild -B -DaltStagingDirectory=${{ github.workspace }}/staging-${{ env.BRANCH_NAME }} -DskipRemoteStaging=true -Dmaven.deploy.skip=true
+
+      #########################################################################
+      # Collect information about build status for central statistics with CI Analytics
+      - uses: camunda/infra-global-github-actions/submit-build-status@main
+        if: always()
+        with:
+          build_status: "${{ contains(steps.*.outcome, 'failure') && 'failure' || 'success' }}"
```

Comment:
👍🏼 How easy would it be also to collect the built time? 🤔 Might be worth to also add as a minimal metric wdyt?

Reply:
Doable but not so easy that's why I didn't do it in the first iteration yet - I was more interested in failure rates.